backend/src/gamestate/mod.rs

```diff
@@ -3,4 +3,5 @@ pub mod player;
 
 pub type WSMessage = String;
 
-pub static CARDS_DEALT_AT_GAME_START: i32 = 7;
+pub static CARDS_DEALT_TO_PLAYERS: usize = 7;
+pub static CARDS_TOTAL_IN_GAME: usize = 108;
```

Comment:
`CARDS_TOTAL_IN_GAME` může velmi snadno neodpovídat realitě - buď na to navaž funkcionalitu balíčku, nebo si to uveď jako konstantu v testech. Takhle to vypadá, že můžeš změnou tohoto čísla změnit počet karet ve hře, což se zcela určitě nestane.

Reply:
True, je to jenom pro testy.

Reply:
Fixed in 8c4bf1ee7b0a5768da4286a9b7afb8999eeb6473.